index.js

```diff
@@ -9,4 +9,4 @@ const sum = (a,b) => {
     return a + b;
 };
 
-sum()
+sum(null,null)
```

Comment:
we cant pass the null args into the sum fnction

Reply:
ok i got i will change it